<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_answer">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>450</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Xa">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Xa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Ya">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ya</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Xb">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Xb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_angl">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Угол</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Yb">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>370</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Yb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_d">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>70</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Горизонтальное проложение</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Xa">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Ya">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_angl">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_d">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_changeTask">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>240</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Перейти к обратной задаче</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Xb_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Yb_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>430</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_changeTask_back">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Перейти к прямой задаче</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Xa_reverse">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Xa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Ya_reverse">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ya</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Xb_reverse">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Xb</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Yb_reverse">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Yb</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Xa_reverse">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Ya_reverse">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Xb_reverse">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Yb_reverse">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>370</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dir_angle">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Угол</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dir_angle_ans">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>420</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>370</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Горизонтальное проложение</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_d_reverse">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
